spec/services/arcgis_api/geocoder_spec.rb

```diff
@@ -46,18 +46,14 @@
     it 'returns candidates from magic_key' do
       stub_request_candidates_response
 
-      suggestions = subject.find_address_candidates('abc123')
-
-      expect(suggestions.first.as_json).to eq(
-        {
-          'address' => '100 Main Ave, La Grande, Oregon, 97850',
-          'location' => { 'longitude' => -118.10754025791812, 'latitude' => 45.328271485226445 },
-          'street_address' => '100 Main Ave',
-          'city' => 'La Grande',
-          'state' => 'OR',
-          'zip_code' => '97850',
-        },
-      )
+      first_candidate = subject.find_address_candidates('abc123').first
+
+      expect(first_candidate.address).to be_present
+      expect(first_candidate.location).to be_present
```

Comment:
Could still be worth checking that it's a hash containing `longtitude` and `latitude`